Mathlib/Data/Nat/Pairing.lean

```diff
@@ -50,23 +50,23 @@ def unpair (n : ℕ) : ℕ × ℕ :=
 #align nat.unpair Nat.unpair
 
 @[simp]
-theorem mkpair_unpair (n : ℕ) : mkpair (unpair n).1 (unpair n).2 = n := by
+theorem mkPair_unpair (n : ℕ) : mkPair (unpair n).1 (unpair n).2 = n := by
```

Comment:
I wonder if the motivation here was to be consistent with `unpair`? Is `mkPair` / `unPair` better or worse than this PR / what we had before?